model/src/test/kotlin/spdx/SpdxExpressionTest.kt

```diff
@@ -26,8 +26,21 @@ import com.here.ort.model.yamlMapper
 import io.kotlintest.shouldBe
 import io.kotlintest.specs.WordSpec
 
+import java.util.EnumSet
+
 class SpdxExpressionTest : WordSpec() {
     init {
+        "spdxLicenses()" should {
+            "contain all valid SPDX licenses" {
+                val expression = "MIT OR (invalid1 AND Apache-2.0 WITH exp) AND (BSD-3-Clause OR invalid2 WITH exp)"
+                val spdxExpression = SpdxExpression.parse(expression)
```

Comment:
I'm a bit confused now. I was expecting the SpdxExpression class to contain a valid expression, e.g. so that as soon as you have an instance of that expression you know that it is valid. This would lead to simpler client code.

This test now successfully parses an ''invalid' expression. 
It's a bit out of scope of this commit, but would it make sense to change 'SpdxExpression.parse' to throw?